### ISSUE TITLE
Container GPU Setup: Could not load library libcublasLt.so.12; Services have not converged

### ISSUE THREAD

**Issue by stauntonjr** [vespa-engine/vespa#27893]
[vespa_logs.txt](https://github.com/vespa-engine/vespa/files/12162863/vespa_logs.txt)
[nvidia-container-toolkit.log](https://github.com/vespa-engine/vespa/files/12162866/nvidia-container-toolkit.log)
[system_buildC.txt](https://github.com/vespa-engine/vespa/files/12162868/system_buildC.txt)
July 25 2023

**Describe the bug**
Vespa (8.198.18) app will not deploy. Logs show unloaded CUDA library although file is on LD_LIBRARY_PATH. 

**To Reproduce**
1. Update Windows, Dell, NVIDIA Graphics driver
2. Allow access to all users to the GPU performance counters in NVIDIA Control Panel Developer Mode
3. Update WSL, Configure default version to WSL2
4. Follow step-by-step instructions in attached file system_buildC.txt
> tl;dr:
> install ubuntu 22.04 in WSL2
> install cuda 12.2 in WSL2
> append relevant directories to PATH and LD_LIBRARY_PATH
> install nvidia-cuda-toolkit
> install docker
> install nvidia-container-toolkit
> generate the "Container Device Interface" config json
> run vespa container and install vespa-onnxruntime-cuda
> try to deploy an app

**Expected behavior**
The app should read LD_LIBRARY_PATH, load libcublasLt.so.12, converge and deploy.

**Environment :**
OS Name:	Microsoft Windows 11 Pro
Version:	10.0.22621 Build 22621
System Model:	XPS 15 9520
Processor:	12th Gen Intel(R) Core(TM) i7-12700H, 2300 Mhz, 14 Core(s), 20 Logical Processor(s)
BIOS Version/Date:	Dell Inc. 1.13.1, 4/17/2023
SMBIOS Version:	3.4
BIOS Mode:	UEFI
Total Physical Memory:	31.7 GB

NVIDIA Studio Driver: 536.67
NVIDIA-SMI: 535.86.01

WSL Linux kernel: 5.15.90.1-microsoft-standard-WSL2
WSL Distro Description:    Ubuntu 22.04.2 LTS

"CUDA Version": 12.2
nvcc: Cuda compilation tools, release 12.2, V12.2.91 Build cuda_12.2.r12.2/compiler.32965470_0

**Vespa version**
[8.198.18](https://hub.docker.com/layers/vespaengine/vespa/8.198.18/images/sha256-f69fa641cf866ed18243a0bd990968bf8bc41a176b4b3dcf32caaa62ac210013?context=explore)

**Comment by stauntonjr** [comment]
[vespa_logs.txt](https://github.com/vespa-engine/vespa/files/12162812/vespa_logs.txt)
[system_buildC.txt](https://github.com/vespa-engine/vespa/files/12162817/system_buildC.txt)
[nvidia-container-toolkit.log](https://github.com/vespa-engine/vespa/files/12162824/nvidia-container-toolkit.log)
